extensions/control-plane/api/management-api/contract-definition-api/src/main/java/org/eclipse/edc/connector/api/management/contractdefinition/model/ContractDefinitionRequestDto.java

```diff
@@ -38,6 +48,8 @@ public class ContractDefinitionRequestDto {
     @Valid
     @NotNull(message = "criteria cannot be null")
     private List<CriterionDto> criteria = new ArrayList<>();
+    @NotNull(message = "validity cannot be null")
+    private DurationDto validity = DEFAULT_VALIDITY;
```

Comment:
I wonder why need the `DurationDto` at all. What value does it add that a `long` would not also give?